compatibility-tests/futures/src/lib.rs

```diff
@@ -61,14 +61,29 @@ async fn load_stock_data_concurrent() -> Result<String, Error> {
     Ok(format!("{}+{}", apple, google))
 }
 
+// Returns of the combinators implement `Future`
+async fn load_stock_data_sequential_again() -> Result<String, Error> {
+    let apple = api::fetch_page("apple")
+        .context(UnableToLoadAppleStock)
+        .await?;
+
+    let google = api::fetch_page("google")
+        .with_context(|| UnableToLoadGoogleStock {
+            name: String::from("sequential"),
+        })
+        .await?;
+
+    Ok(format!("{}+{}", apple, google))
+}
+
 // Can be used as a `Stream` combinator
 async fn load_stock_data_series() -> Result<String, Error> {
     let apple = api::keep_fetching_page("apple").context(UnableToLoadAppleStock);
     let google = api::keep_fetching_page("google").with_context(|| UnableToLoadGoogleStock {
         name: String::from("stream"),
     });
 
-    let together = apple.into_stream().zip(google.into_stream());
```

Comment:
I'm not following why this change was made; was it *required* to change or is it simply clearer to change it?

Reply:
Ah, right. `into_stream` is the hack needed to convert a `TryStream` back into a `Stream`, which we don't need anymore.

However, leaving the method calls *would* still work, since we implicitly implement `TryStream`.